app/websocket/src/main/kotlin/com/xorker/draw/websocket/WebSocketRouter.kt

```diff
@@ -31,6 +35,14 @@ class WebSocketRouter(
 
             RequestAction.DRAW -> mafiaGameUseCase.draw(session.getDto(), request.extractBody())
             RequestAction.END_TURN -> mafiaGameUseCase.nextTurnByUser(session.getDto())
+
+            RequestAction.VOTE -> {
+                val requestDto = request.extractBody<VoteMafiaRequest>()
+                val userId = UserId(requestDto.userId)
+                val sessionDto = sessionUseCase.getSession(SessionId(session.id)) ?: throw InvalidRequestValueException
```

Comment:
p3. 계속 쓰이는 조회 코드라서 확장함수로 만들어뒀습니다.

```suggestion
                val sessionDto = session.getDto()
```